CommonTools/ParticleFlow/plugins/PFnoPileUpPacked.cc

```diff
@@ -0,0 +1,91 @@
+// system include files
+#include <memory>
+#include <string>
+
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "DataFormats/Candidate/interface/Candidate.h"
+#include "DataFormats/VertexReco/interface/VertexFwd.h"
+#include "DataFormats/Common/interface/Association.h"
+
+#include "DataFormats/VertexReco/interface/Vertex.h"
+
+#include "FWCore/Utilities/interface/Exception.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+
+using namespace std;
+using namespace edm;
+using namespace reco;
+
+/**\class PFnoPileUpPacked
+\brief Identifies pile-up candidates from a collection of Candidates, and
+produces the corresponding collection of NoPileUpCandidates.
+
+\author Andreas Hinzmann
+\date   May 2021
+
+*/
+
+class PFnoPileUpPacked : public edm::stream::EDProducer<> {
```

Comment:
In the class and filename:
```suggestion
class PFNoPileUpPacked : public edm::stream::EDProducer<> {
```

for consistency
```
$ find . -name "*PFno*"
$ find . -name "*PFNo*"
./PhysicsTools/PatUtils/plugins/ShiftedPFCandidateProducerForPFNoPUMEt.h
./PhysicsTools/PatUtils/plugins/ShiftedPFCandidateProducerForPFNoPUMEt.cc

```